packages/bahmni-design-system/src/molecules/textAreaWClose/styles/TextAreaWClose.module.scss

```
@@ -46,3 +32,7 @@
     color: $icon-primary !important;
   }
 }
+
+:global(.cds--text-area) {
```

Stylelint (17.11.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)